lisa_commands/request_helper.py

```diff
@@ -127,6 +127,29 @@ def youtube_search(query):
         return f"https://www.youtube.com/watch?v={video_id}"
     else:
         return None
+
+def wikipedia_search(query):
+    """
+    Submit a search to wikipedia
+    :query what to query for
+    """
+    request_params = {
```

Comment:
to handle spaces in our query these params need to be encoded like this:
    request_params = urllib.parse.urlencode({
    'action':'query',
    'list':'search',
    'srsearch': query,
    'format':'json'
    })